gate-teamware/README.md

```diff
@@ -57,6 +57,15 @@ Things you will commonly need to override include:
   - `replicaCount` (default 1) - the number of replicas of the Django container to run.  Alternatively you can set `backend.autoscaling.enabled` to `true` for auto-scaling based on CPU usage
 - `staticFiles`
   - `replicaCount` (default 1) - the same for the static files nginx, though this is highly unlikely to need more than one replica as it's a simple static file server
+- `privacyPolicy` - contact details of the service's host and administrator to be rendered in the privacy policy and terms & conditions.
+  - `host` - the organisation or individual responsible for managing the deployment of the teamware instance.
+    - `name` - host organisation/individual's name.
+    - `address` - physical address.
+    - `contact` - a means of contact, supports HTML for e.g. email or contact form links.
+  - `admin` - the individual or organisation responsible for managing users of the teamware instance.
```

Comment:
Re my comment on the deployment template - if we go the route of defaulting the admin options to the host ones then

```
- `admin` - the individual or organisation responsible for managing users of the teamware instance, if this is not the same as the host.
```